modules/invoicein/view/list.php

```diff
@@ -180,8 +180,17 @@
            print $_lib['format']->Amount($InvoiceO->TotalCustPrice);
         ?>
       </td>
-      <td><? print $InvoiceO->Department ?></td>
-      <td><? print $InvoiceO->Project ?></td>
+      <?
```

Comment:
Maybe move this to respective classes as methods instead of doing the query here just call the method.